spec/beaker/host/cisco_spec.rb

```diff
@@ -14,10 +14,10 @@ module Cisco
 
     describe '#prepend_commands' do
 
-      context 'for cisco-5' do
+      context 'for cisco-nexus' do
 
         before :each do
-          @platform = 'cisco-5-x86'
+          @platform = 'cisco-nexus-x86'
```

Comment:
Well, at this point we don't need different commands for a particular version - we can treat all 'cisco_nexus-XXXXX' as equivalent. I'm just trying to think forward-compatibility here in case something does change.